compiler/next/include/chpl/queries/Context.h

```diff
@@ -199,9 +199,9 @@ class Context {
 
   std::vector<const querydetail::QueryMapResultBase*> queryStack;
 
-
   querydetail::RevisionNumber currentRevisionNumber = 1;
   bool enableDebugTracing = false;
+  int numQueriesRunThisRevision_ = 0;
```

Comment:
What does revision mean in this context?

Reply:
See my other answers